WordPress/Classes/ViewRelated/NUX/SiteCreationDomainSearchTableViewCell.xib

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a bit nitpicky, but the icon doesn't line up with the text in the header & search results like it does in the design.